### ISSUE TITLE
Allow making comment a reply after posting it

### ISSUE THREAD

**Issue by eliduvid** [MoeraOrg/moera-issues#344]
If you forgot to click reply button on a comment you're replying to? there's no way to make it one afterwards. I think it should be possible :)

**Comment by smelamud** [comment]
Yes, but I added this limitation because making the reply reference bound to a version of the comment instead of the comment itself makes signing and signature verification more complex. We will need to take into account the timestamp when the reply reference was created (it differs from the timestamp of the version itself, because it may be created in one of previous versions), fetch that particular revision of the referred comment and use its hash for signing/verification. And the referred comment may also have a reply reference and we will need to verify it also in respect of the particular version at that particular moment.

I don't say this is impossible, but requires much more effort than it seems.
